docs/how-to-guides/jdbc-cosmos-notes.md

```diff
@@ -102,4 +102,46 @@ db_client = client.get_database_client(some_cosmosdb_database)
 container_client = db_client.get_container_client(some_cosmosdb_collection)
 doc = container_client.read_item(some_key)
 feature_value = doc['feature_name']
-```
\ No newline at end of file
+```
+
+## Using ElasticSearch as online store
```

Comment:
Maybe we should put a "Beta" or "preview" here to lower the expectation?

Reply:
This should have the same quality as CosmosDb, they share exactly the same underlying code, the only difference is the user interaction part in the Feathr client, which is merely a wrapper.